vendor/riscv_isa_sim.lock.hjson

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is generated by the util/vendor script. Please do not modify it
+// manually.
+
+{
+  upstream:
+  {
+    url: https://github.com/joxie/riscv-isa-sim
+    rev: c2186bf1731b2a123ccc785ce9585861d370886f
```

Comment:
As we only want the tests out of this and nothing else let's just vendor in those and cut out the rest. This is what we do with the 'lowrisc_ip' directory which is actually vendoring in part of OpenTitan see https://github.com/lowRISC/ibex/blob/master/vendor/lowrisc_ip.vendor.hjson to see how it works.

Reply:
Thanks for pointing this. I have updated this to only include the needed files/directories.

```
tests/mseccfg
arch_test_target
LICENSE
```